tdrs-backend/gunicorn.py

```diff
@@ -0,0 +1,24 @@
+"""Gunicorn development config file."""
+
+# WSGI application path MODULE_NAME:VARIABLE_NAME
+wsgi_app = "tdpservice.wsgi:application"
+
+# The granularity of Error log outputs
+# loglevel = "debug"
```

Comment:
What would be our expected output level? `INFO`? Do we disregard the `LOG_LEVEL` env var?

Reply:
I see that log-level is still used in the gunicorn command but this line is left intact in the "dev" version of this file. Would that create a conflict?

Reply:
It is unset because it was polluting production logs. Will have to reset it.

For dev, we definitely set it to "debug" and for production it should be set to either "info" or "warning".

we can also change the log format (https://docs.gunicorn.org/en/stable/settings.html#logging), but we shouldn't really need to since we are getting all the information from Django log.